TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
```

Comment:
I've more typically seen vars initialized/set in `viewDidLoad`. What was the motivation for initializing it here?